cpp/tests/copying/get_value_tests.cu

```diff
@@ -0,0 +1,159 @@
+/*
```

Comment:
Should this file be removed for this PR now that get_element is in a separate PR?